tests/main.nf.test

```diff
@@ -29,6 +30,67 @@ nextflow_pipeline {
             }
         }
 
+        then {
+            assertAll(
+                { assert workflow.success }
+            )
+        }
+    }
+    test("Run with profile test somatic") {
+
+        when {
+            params {
+                max_cpus            = 2
+                max_memory          = '6.GB'
+                max_time            = '6.h'
+                input               = 'https://raw.githubusercontent.com/kubranarci/benchmark_datasets/main/sarek_full_test_somatic_v3.4.2/samplesheet_somatic.csv'
+                outdir              = 'results'
+                genome              = 'GRCh38'
+                analysis            = 'somatic'
+                sample              = "SEQC2"
+                method              = 'sompy,truvari'
+                preprocess          = "filter_contigs"
+                truth_snv           = "https://raw.githubusercontent.com/kubranarci/benchmark_datasets/main/somatic/truth/hg38/sSNV_truth_set_v1.0.chr21.vcf.gz"
+                high_conf_snv       = "https://raw.githubusercontent.com/kubranarci/benchmark_datasets/main/somatic/truth/hg38/high-confidence_sSNV_in_HC_regions_v1.2.chr21.vcf.gz"
+                truth_indel         = "https://raw.githubusercontent.com/kubranarci/benchmark_datasets/main/somatic/truth/hg38/sINDEL_truth_set_v1.0.chr21.vcf.gz"
+                high_conf_indel     = "https://raw.githubusercontent.com/kubranarci/benchmark_datasets/main/somatic/truth/hg38/high-confidence_sINDEL_in_HC_regions_v1.2.chr21.vcf.gz"
+                truth_sv            = "https://raw.githubusercontent.com/kubranarci/benchmark_datasets/main/somatic/truth/hg38/13059_2022_2816_MOESM4_ESM.vcf.gz"
+                itruth_ignore       = true
```

Comment:
I think you just need outdir as the rest is in your test profile

Reply:
Then, I should add config as

`config "../conf/test.config" `

?